### DISCUSSION TITLE
App version mismatch (extras/picpick)

### DISCUSSION THREAD

**Discussion by max7pro** [ScoopInstaller/Scoop#5664] [Help and Questions]
Hello
App "Picpick" form "extras" bucket listed with scoop list/search/info as version 7.2.2. In fact it installed as version 7.2.0 (and app after after it was started also asks to download new version).

![image](https://github.com/ScoopInstaller/Scoop/assets/145686781/30d05c38-ba06-428d-9495-87cc36123fa5)

Thank you for you help and for your scoop service, it is my favorite method to install paps.

**Comment by HUMORCE** [comment]
https://github.com/ScoopInstaller/Extras/commit/551eb44a11bd5c1fda4f76035619142d02ddf207
